src/main/kotlin/mate/academy/service/FigureSupplier.kt

```diff
@@ -0,0 +1,49 @@
+package mate.academy.service
+
+import mate.academy.model.Color
+import mate.academy.model.Figure
+import mate.academy.model.figures.Circle
+import mate.academy.model.figures.IsoscelesTrapezoid
+import mate.academy.model.figures.Rectangle
+import mate.academy.model.figures.RightTriangle
+import mate.academy.model.figures.Square
+import kotlin.random.Random
+
+private const val RANGE = 100.0
+private const val DEFAULT_RADIUS = 10.0
+
+class FigureSupplier {
+    private val colorSupplier = ColorSupplier()
+
+    fun getRandomFigure(): Figure {
+        val figures: Array<Figure> = arrayOf(
```

Comment:
Review this method. The solution is working, but every time it is called, array of objects is created)
Try to think of a solution where you choose a figure randomly first, and then create the object